Networking/Networking/Network/AlamofireNetwork.swift

```diff
@@ -107,16 +107,11 @@ public class AlamofireNetwork: Network {
                                         to request: URLRequestConvertible,
                                         completion: @escaping (Data?, Error?) -> Void) {
         let request = requestConverter.convert(request)
-        sessionManager.upload(multipartFormData: multipartFormData, with: request) { (encodingResult) in
-            switch encodingResult {
-            case .success(let upload, _, _):
-                upload.responseData { response in
-                    completion(response.value, response.error)
-                }
-            case .failure(let error):
-                completion(nil, error)
+        sessionManager
+            .upload(multipartFormData: multipartFormData, with: request)
+            .responseData { response in
+                completion(response.value, response.error)
             }
-        }
```

Comment:
From [the migration guide](https://github.com/Alamofire/Alamofire/blob/723fa5a6c65812aec4a0d7cc432ee198883b6e00/Documentation/Alamofire%205.0%20Migration%20Guide.md):
> MultipartFormData’s API has changed and the top level upload methods to create and upload MultipartFormData have been updated to match other request APIs, so it’s not longer necessary to deal with the Result of the multipart encoding.